package.json

```diff
@@ -22,15 +22,13 @@
         ]
     },
     "devDependencies": {
+        "@concordium/web-sdk": "^7.0.0",
```

Comment:
It's there to ensure that the peer dependency is installed while developing. Is there a better way to do this?

Reply:
I'm just assuming that having it as a peer dependency takes care of that. We also have react as a peer dependency and don't need to add that here. Are you sure it doesn't just work without it?

Reply:
It works because the peer dependencies are installed in the root `node_modules` from having them in dependencies of the sample apps.

I think it's nice to have them as a dev dependency in the workspace, as that will make sure it keeps working even if the samples no longer depend on the peer dependencies (theoretical issue, I know...)

Reply:
I would prefer not addressing that issue until it's solving an actual problem. For now it's just duplication that we have to remember to keep in sync.

Reply:
Removed it.